3_vizualize/src/plot_data.R

```diff
@@ -0,0 +1,44 @@
+plot_data <- function(data_in){
+  # Create a plot
+  png(file = file.path('3_vizualize/out/figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
It is helpful to have arguments for commonly edited plot features, like output save path, plot axes, plot dimensions, etc.

Reply:
Great suggestion. I pulled out the internal `png` arguments using the `...` argument. I also added more detailed documentation (which I had accidentally left out).

I took a slightly different approach to pulling out the other plot details such as `col`, `pch` and the axes:

* I did not add an argument for specifying the x-axis. Instead I removed the hard coded x-axis values and instead opted to  to filter for all unique values of `n_prof`. 
* I did not add arguments for `pch` or `col`. These values are arguments to `prep_data` therefore it seemed redundant to call them again in `plot_data` (i.e., why would these change between processing and plotting?). Instead, i used the input data (`data_in`) to generate a list of values that are then used to generate the legend in the upper left corner.